geode-core/src/distributedTest/java/org/apache/geode/internal/cache/PartitionedRegionSingleHopDUnitTest.java

```diff
@@ -386,75 +428,70 @@ public void testSingleHopWithHA() {
 
     // again update
     for (int i = 1; i <= 16; i++) {
-      testRegion.put(i, i + 10);
+      partitionedRegion.put(i, i + 10);
     }
   }
 
   @Test
   public void testSingleHopWithHAWithLocator() {
-    int port3 = AvailablePort.getRandomAvailablePort(AvailablePort.SOCKET);
-    String locator = "localhost[" + port3 + "]";
+    int locatorPort = vm3.invoke(() -> startLocator());
```

Comment:
dunitrunner revealed intermittent failures in testSingleHopWithHAWithLocator with an underlying BindException as the cause.

The easiest fix is to change startLocator to use port zero and then return whatever port the Locator ended up binding to.